samples/openthread/libraries/CMakeLists.txt

```diff
@@ -0,0 +1,71 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+cmake_minimum_required(VERSION 3.13.1)
+
+include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
+project(NONE)
+
+FILE(GLOB app_sources src/*.c)
+
+target_sources(app PRIVATE ${app_sources})
+
+execute_process(
+  COMMAND           git rev-parse HEAD
+  WORKING_DIRECTORY ${openthread_SOURCE_DIR}
+  RESULT_VARIABLE   git_return
+  OUTPUT_VARIABLE   openthread_git_hash)
+
+list(APPEND OT_SETTINGS "OpenThread_commit=${openthread_git_hash}\n\n")
+
+# Store all OT related variables
+get_cmake_property(_variableNames VARIABLES)
+list (SORT _variableNames)
+foreach (_variableName ${_variableNames})
+    if("${_variableName}" MATCHES "^CONFIG_OPENTHREAD_.*|^OT_.*") 
+	    list(APPEND OT_SETTINGS "${_variableName}=${${_variableName}}\n")
+    endif()
+endforeach()
+
+FILE(WRITE "${NRFXLIB_DIR}/openthread/openthread_lib_configuration.txt" ${OT_SETTINGS})
+
+set(OPENTHREAD_LIBS
+	"${PROJECT_BINARY_DIR}/modules/openthread/build/src/core/libopenthread-ftd.a"
+	"${PROJECT_BINARY_DIR}/modules/openthread/build/src/core/libopenthread-mtd.a"
+	"${PROJECT_BINARY_DIR}/modules/openthread/build/src/core/libopenthread-radio.a"
+	"${PROJECT_BINARY_DIR}/modules/openthread/build/src/cli/libopenthread-cli-ftd.a"
+	"${PROJECT_BINARY_DIR}/modules/openthread/build/src/cli/libopenthread-cli-mtd.a"
+	"${PROJECT_BINARY_DIR}/modules/openthread/build/src/ncp/libopenthread-ncp-ftd.a"
+	"${PROJECT_BINARY_DIR}/modules/openthread/build/src/ncp/libopenthread-ncp-mtd.a"
+	"${PROJECT_BINARY_DIR}/modules/openthread/build/src/ncp/libopenthread-rcp.a"
+)
+
+message(STATUS "OPENTHREAD_LIBS=${OPENTHREAD_LIBS}")
+
+#TODO: is it true?
```

Comment:
It does not seem correct.

Reply:
TBH I was thinking more of using nrfxlib function nrfxlib_calculate_lib_path(). This shall return platform path for given configuration.

Reply:
Thanks, I will take a look at it.

Reply:
Also I was hoping that most of this code will go to https://github.com/nrfconnect/sdk-nrfxlib/blob/0c49b2d734122c086c260e16a45e29efa4f40c71/openthread/CMakeLists.txt file, but sadly it is not merged yet.